p/pywt/build_info.json

```diff
@@ -10,7 +10,10 @@
   "docker_build": false,
   "validate_build_script": true,
   "use_non_root_user": "false",
-  "*": {
+  "1.4.*": {
```

Comment:
@meetj15  update the maintainer name and map v1.8.0 to  "pywt_1.8.0_ubi_9.5.sh" and map * to "pywt_ubi_9.3.sh" at end of the json file

Reply:
@meetj15 add the default script which is pywt_ubi_9.3.sh at last